AlphaWallet/Market/UniversalLinkHandler.swift

```diff
@@ -46,10 +46,9 @@ extension Array {
 
 public class UniversalLinkHandler {
 
-    public let urlPrefix = "https://app.awallet.io/"
-
     //message is with 32 bytes each of price and expiry and is shortened for link
     func createUniversalLink(signedOrder: SignedOrder) -> String {
+        let urlPrefix = Config().magicLinkPrefix.description
```

Comment:
Might be good to hardcode to use the legacy "https://app.awallet.io/" when creating for mainnet for compatibility for existing app versions? (we can't change the `magicLinkPrefix` value because it's used elsewhere). If so create an issue so we don't forget to remove it?

Reply:
I think it best to simply move on